pkg/v1/cli/component/output.go

```diff
@@ -14,8 +14,10 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-const colWidth = 300
-const indentation = `  `
+const (
+	colWidth    = 80
```

Comment:
Some guides recommend to set it to 79 (just to be on safe side?) - for example [pep8](https://peps.python.org/pep-0008/#maximum-line-length)

80 is fine as well I think.

Reply:
I agree. If we want to be concerned about 80 (or 79) characters, then we should probably consider the entire line of output and not just the one column. So the limit would end up being more like the 30-50 range like you suggest.

I think this at least makes it a reasonable width, so I think this is definite improvement.